.kokoro/tests/run_single_test.sh

```diff
@@ -41,6 +41,19 @@ fi
 nox -s "$RUN_TESTS_SESSION"
 EXIT=$?
 
+# Inject region tag data into the test log
+REGION_TAG_PARSER_DIR="$PARENT_DIR/region-tag-parser"
```

Comment:
Can you use a temporary directory? Also it's great if you make sure the temporary directory will be deleted after the test.

Reply:
Update: maybe you can just install them in `/opt` or somewhere then it will only exists in the container.

`/tmp` is shared between the container and the host for a reason, so it's probably better to use somewhere else.